### PR TITLE
add #BMP list

### PR DESCRIPTION
**Pull request by NS-Kazuki**

### CODE REVIEW

File: iBMSC/ChartIO.vb [hunk 1]
```diff
@@ -375,10 +383,14 @@ AddExpansion:       xExpansion &= sLine & vbCrLf
             If Not hWAV(i) = "" Then xStrHeader &= "#WAV" & C10to36(i) &
                                                     " " & hWAV(i) & vbCrLf
         Next
+        For i = 1 To UBound(hBMP)
+            If Not hBMP(i) = "" Then xStrHeader &= "#BMP" & C10to36(i) &
```
Comment (zardoru) [inline iBMSC/ChartIO.vb:387]: No Base-16 support?

File: iBMSC/MainWindow.vb [hunk 2]
```diff
@@ -1881,6 +1892,47 @@ EndSearch:
         End Select
     End Sub
 
+    Private Sub ValidateBmpListView()
```
Comment (zardoru) [inline iBMSC/MainWindow.vb]: This seems unused?

**Review by zardoru — COMMENTED** [review]
A few minor observations.
μBMSC_2019-07-25_16-18-17
The BPM column displays the #WAV label instead of the #BPM label on the column. I think it would be a good idea to fix this.

If you have no time, let me know, this is good to be merged in any case.

**Comment by NS-Kazuki** [comment]
Good idea! and fix it.

**Comment by zardoru** [comment]
Fantastic job!

**Pull request merged**